apps/server/src/adapters/OscAdapter.ts

```diff
@@ -16,14 +16,14 @@ export class OscServer implements IAdapter {
     this.osc.on('error', (error) => logger.error(LogOrigin.Rx, `OSC IN: ${error}`));
 
     this.osc.on('message', (msg) => {
-      // message should look like /ontime/{path}/{params?} {args} where
+      // message should look like /ontime/{type}/{params?} {args} where
```

Comment:
I am not sure about renaming this to `type`
`type` is a loaded word with a lot of meaning, none of which relevant in this case

if you dont like path, how about `command`?

Reply:
just wanted it to match the `WS` `{type: , payload:}`

but command or action is also ok for me